### ISSUE TITLE
Adding Entities recognition using haptik (Feature suggestion)

### ISSUE THREAD

**Issue by munzzz5** [nat/natbot#4]
Hello, Im amazed at this implementation of GPT-3. 
I would love to add an extra later to theprompt by preprocessing the text and adding entities with metalabel while passing prompt to GPT-3

An example would be:
Buy [Brand] Adidas shoes from Amazon [company] India [location] under $200 [numeric] 

We could achieve this using the hello haptik repository (link: https://github.com/hellohaptik/chatbot_ner) 

Im not completely sure if this will help the output. What do you think?

Kudos!

**Comment by lee101** [comment]
i'm just guessing but i personally dont think annotating using entity types would help in terms of gpt-3 understanding, because it wasn't trained on this NER style, unless you do specific things with the detected entities unrelated to gpt-3 e.g. handle the location in a specific way via google map search and pull out the location for it etc.

**Comment by jordn** [comment]
Yeah I wouldn't expect this to help either. The NER model will almost certainly be worse at understanding the words than GPT-3, so it would only add some noise and extra tokens.
